pkg/osquery/extension.go

```diff
@@ -160,65 +146,60 @@ func NewExtension(ctx context.Context, client service.KolideService, k types.Kna
 		return nil, fmt.Errorf("setting up agent keys: %w", err)
 	}
 
-	identifier, err := IdentifierFromDB(configStore)
-	if err != nil {
-		return nil, fmt.Errorf("get host identifier from db when creating new extension: %w", err)
-	}
-
 	nodekey, err := NodeKey(configStore)
 	if err != nil {
-		k.Slogger().Log(context.TODO(), slog.LevelDebug,
+		slogger.Log(context.TODO(), slog.LevelDebug,
```

Comment:
not blocking but should we be using the parent context for these? it looks like there is one available from func args

Reply:
Ah whoops, thank you! I will update